.github/workflows/cypress.yml

```
@@ -79,3 +79,96 @@ jobs:
         with:
           name: cypress-screenshot-artifacts
           path: cypress/screenshots
+
+      - name: Archive Cypress Mochawesome test results
+        uses: actions/upload-artifact@v2
+        if: ${{ always() }}
+        with:
+          name: cypress-mochawesome-test-results-artifact-${{ matrix.ci_node_index }}
+          path: cypress/results
+
+  mochawesome-report:
+    if: ${{ always() }}
+    needs: [build]
```

When there's only one dependency, you can omit the brackets.

```suggestion
    needs: build
```

---

Done. Thanks Eugene.